### DISCUSSION TITLE
Keep route URI as title in sidebar navigation

### DISCUSSION THREAD

**Discussion by tklie** [dedoc/scramble#93] [Ideas]
Hi, thanks for the great package!

When adding a 1- or 2-line DocBlock with title and/or description to a controller method, the title then is also used in the sidebar navigation instead of the route's URI. This makes it very difficult for my frontend devs to actually find the documentation of the route they are looking for.

It would be great if we had a config option to disable this behavior and keep the URI in the sidebar.

**Comment by romalytvynenko** [comment]
Hey @tklie 

Thanks for the suggestion.

Scramble uses Stoplight Elements UI to show API documentation and this is why we cannot actually do some meaningful changes to the UI part (at least for now).

I already have a plan for own UI package with all the suggestions I've already heard 😄 

So until then unfortunately we cannot do any real changes to the UI.

**Reply by tklie** [reply]
Thanks for the feedback! Sounds good :)
